usr/src/cmd/bhyve/Makefile

```diff
@@ -150,8 +155,6 @@ clean: $(SUBDIRS)
 clobber: clean $(SUBDIRS)
 	$(RM) $(CLOBBERFILES)
 
-lint:	lint_SRCS $(SUBDIRS)
```

Comment:
Why is this removed?

Reply:
We've stopped using lint some time ago and use smatch in it's place now. Since it'd be a massive change to go through and remove all the now unused targets, the convention has been to remove them as you touch the makefiles.  Smatch is run as a shadow compiler, so it doesn't need any special makefile rules